tools/lint.py

```diff
@@ -49,5 +50,20 @@ def pylint():
         quiet=True)
 
 
+def clippy():
+    print "clippy"
+    run([
+        "cargo",
+        "clippy",
+        "--all-targets",
+        "--locked",
+        "--",
+        "-D",
+        "clippy::all",
+    ],
+        shell=False,
+        quiet=True)
```

Comment:
I'm a little concerned that this might build the wrong variant (debug, when you're working on release).

We have some environmental variables to deal with this in tools/util.py
https://github.com/denoland/deno/blob/master/tools/util.py#L206-L221
Could you make use of them here?

Reply:
Using `def build_mode(default="debug"):` will always return `"debug"` unless:

- DENO_BUILD_MODE is set
- lint.py is ran with --release (should this be allowed with lint.py?)

Meaning if someone is working in release and runs `lint.py` without meeting any of the above conditions it will default to debug. Should we go with this or try harder to find the current working build **(Maybe listing /target and searching for debug/release folders?)**.